### ISSUE TITLE
Resource: a11y: ARIA describedby has invalid ID

### ISSUE THREAD

**Issue by vyncent-t** [headlamp-k8s/headlamp#1391]
## Description
React AXE error for StatusLabel component reads as followed:

`"ARIA attribute element ID does not exist on the page: aria-describedby=\"container-state-message-coredns\""`

### Screenshot
![React AXE Screenshot of Error](https://github.com/headlamp-k8s/headlamp/assets/78232183/708fd279-c021-47d0-a459-b07a85152e48)

### Criteria / Impact

To reproduce this:
- Start the browser with the command `REACT_APP_SKIP_A11Y=false make run-frontend`
- Navigate to any pods detail page
- Notice prompt and open console inspection
- Reference screenshot for exact location

**Impact**
This could cause issues for screenreaders for missing describe fields
